raster_loader/io.py

```diff
@@ -1352,45 +1058,31 @@ def get_block_dims(file_path: str) -> tuple:
         return raster_dataset.block_shapes[0]
 
 
-def inject_areas_query(raster_table: str, is_quadbin: bool) -> str:
-    location_column = "quadbin" if is_quadbin else "geog"
-    from_metadata_source = f"FROM `{raster_table}` WHERE {location_column} IS NULL"
-    from_blocks_source = f"FROM `{raster_table}` WHERE {location_column} IS NOT NULL"
-    if is_quadbin:
-        block_y = "'$.y'"
-        block_x = "'$.x'"
-        avg_pixel_area_query = f"""
-            SELECT
-              AVG(
-                CAST(JSON_VALUE(attrs, '$.block_area') AS FLOAT64)
-                / (block_height*block_width)
-              )
-            {from_blocks_source}
-        """
-        area_query = f"""
-            SELECT SUM(CAST(JSON_VALUE(attrs, '$.block_area') AS FLOAT64))
-            {from_blocks_source}
-        """
+def inject_areas_query(raster_table: str) -> str:
```

Comment:
I think this can be completely removed. The new metadata does not include the area